src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -701,6 +715,19 @@ private GroupManager getGroupManager(final IUserGroupPersistenceManager userGrou
         return groupManager;
     }
 
+
+    @Inject
+    @Provides
+    @Singleton
+    private IGroupObserver getGroupObserver(EmailManager emailManager, GroupManager groupManager, UserAccountManager userManager,
```

Comment:
As noted above, this was bound as an `.asEagerSingleton()` in the Isaac Guice config, which was only used once. Since (at least for now) we make 9 Segue Guice config modules on startup, this would create 9 of these, which is not what "Singleton" is supposed to mean. Instead move it to a provider, and make the Application request one and store it as a singleton there.